luda/editor/client/src/timeline/clip/renderClip.ts

```diff
@@ -0,0 +1,42 @@
+import { Render } from "namui";
+import { isCameraClip, isSubtitleClip } from "../../clipTypeGuard";
+import { Clip } from "../../type";
+import { CameraTrackSash } from "../cameraTrack/CameraTrackSash";
+import { TimelineState } from "../type";
+import { ClipComponent } from "./ClipComponent";
+import { SubtitleClipComponent } from "./SubtitleClipComponent";
+
+export const renderClip: Render<
+  {
+    timelineState: TimelineState;
+    clip: Clip;
+  },
+  {
+    height: number;
+    maxRight: number;
+  }
+> = (state, props) => {
+  const { clip } = state;
+
+  if (isCameraClip(clip)) {
+    console.log("camera");
```

Comment:
oupppss

Reply:
I removed it. thx ;)
with https://github.com/NamseEnt/namseent/pull/18/commits/cfdc376d5fbe7f729582699affe87cb6a8c52e5f